src/include/network/services/routing.rb

```diff
@@ -258,7 +271,7 @@ def RoutingEditDialog(id, entry, devs)
         end
         route = Builtins.add(route, val)
         val = Convert.to_string(UI.QueryWidget(Id(:genmask), :Value))
-        if val != "-" && val != "0.0.0.0" && !Netmask.Check(val)
+        if val != "-" && val != "0.0.0.0" && !valid_netmask?(val)
```

Comment:
How am I supposed to know the difference between "check the netmask" and "is the netmask valid"?
Why are we not using/fixing the Netmask library instead? Observe these:
- http://www.rubydoc.info/github/yast/yast-yast2/Yast/NetmaskClass
- http://www.rubydoc.info/github/yast/yast-network/Yast/NetworkLanAddressInclude:ValidateNetmask
- http://www.rubydoc.info/github/yast/yast-network/Yast/NetworkRoutinesInclude:validPrefixOrNetmask